.dockerignore

```diff
@@ -0,0 +1,19 @@
+# everything from .gitignore
+image.*
+video.mp4
+video.webm
+cache/
+__pycache__/
+.idea/
+**/.pytest_cache/
+venv/
+.vscode/
+.sass-cache/
+flask_session/
+.DS_Store
+site_settings.json
```

Comment:
`site_settings.json` is in `.gitignore` but an older version is still in the repo
```sh
$ git show origin/frost:site_settings.json
{"Bots": true, "Fart mode": false, "Read-only mode": false, "Signups": true}
```
Not sure whether removing this file from docker entirely will have negative effects